server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -294,19 +302,24 @@ public ImmutableList<AnswerData> getSummaryDataOnlyHidden() {
     // TODO: We need to be able to use this on the admin side with admin-specific l10n.
     ImmutableList.Builder<AnswerData> builder = new ImmutableList.Builder<>();
     ImmutableList<Block> hiddenBlocks = getAllHiddenBlocks();
-    addDataToBuilder(hiddenBlocks, builder, /* hiddenBlocks= */ true);
+    addDataToBuilder(hiddenBlocks, builder, /* showAnswerText= */ false);
     return builder.build();
   }
 
   /**
    * Helper method for {@link ReadOnlyApplicantProgramServiceImpl#getSummaryDataOnlyActive()} and
    * {@link ReadOnlyApplicantProgramServiceImpl#getSummaryDataOnlyHidden()}. Adds {@link AnswerData}
    * data to {@link ImmutableList.Builder<AnswerData>}.
+   *
+   * @param blocks the blocks to add to the builder
+   * @param builder the builder to add the blocks to
+   * @param showAnswerText whether to include the answer text in the result. If {@code false},
+   *     answers are replaced with "N/A".
    */
   private void addDataToBuilder(
       ImmutableList<Block> blocks,
       ImmutableList.Builder<AnswerData> builder,
-      boolean hiddenBlocks) {
+      boolean showAnswerText) {
```

Comment:
I changed the name of this and flipped the boolean's value to better match what it's doing